pyasic/miners/types/whatsminer/M3X/M33.py

```diff
@@ -24,10 +24,7 @@ def __init__(self, ip: str, api_ver: str = "0.0.0"):
         super().__init__(ip, api_ver)
         self.ip = ip
         self.model = "M33 V10"
-        self.nominal_chips = 0
-        warnings.warn(
-            "Unknown chip count for miner type M33V10, please open an issue on GitHub (https://github.com/UpstreamData/pyasic)."
-        )
+        self.nominal_chips = 33
```

Comment:
Just make sure this is 3 boards not 4, same for the rest, M33s on other model seem to have 4.